src/features/goblins/bank/components/Deposit.tsx

```diff
@@ -336,24 +336,13 @@ const DepositOptions: React.FC<Props> = ({
     sflBalance.eq(0);
   const validDepositAmount = sflDepositAmount > 0 && !amountGreaterThanBalance;
 
-  // if (!canDeposit) {
```

Comment:
I think u forgot to uncomment this before pushing

Reply:
It was commented out, I just removed it